CI failing for ``test_repartition_quantiles``

`dask_expr/tests/test_quantiles.py::test_repartition_quantiles` is failing locally for me in `main`. The failure is also showing up in CI:

```
E               AssertionError: Series.index are different
E               
E               Series.index values are different (16.66667 %)
E               [left]:  Index([0.0, 0.2, 0.4, 0.6000000000000001, 0.8, 1.0], dtype='float64')
E               [right]: Index([0.0, 0.2, 0.4, 0.6, 0.8, 1.0], dtype='float64')
```

--- a/dask_expr/tests/test_quantiles.py
+++ b/dask_expr/tests/test_quantiles.py
@@ -12,8 +12,8 @@ def test_repartition_quantiles():
     expected = pd.Series(
         [1, 1, 3, 7, 9, 15], index=[0, 0.2, 0.4, 0.6, 0.8, 1], name="a"
     )
-    assert_eq(result, expected)
+    assert_eq(result, expected, check_exact=False)
 
     result = df.a._repartition_quantiles(npartitions=4)
     expected = pd.Series([1, 2, 5, 8, 15], index=[0, 0.25, 0.5, 0.75, 1], name="a")
-    assert_eq(result, expected)
+    assert_eq(result, expected, check_exact=False)

